changelogs/current.yaml

```diff
@@ -14,4 +14,8 @@ removed_config_or_runtime:
 
 new_features:
 
+- area: http
+  change: |
+    made :ref:`allow_coalesced_connections <envoy_v3_api_field_extensions.clusters.dynamic_forward_proxy.v3.ClusterConfig.allow_coalesced_connections>` visible, enabling connection coalescing for HTTP/2 and HTTP/3 dynamic forward proxy connections.
```

Comment:
nit: This is the first time the user has seen this, so can you write it from the perspective of it being a new feature?